joomla-projects/media-manager-improvement
# Basic file upload

This PR adds the basic file upload feature. 

# Testing instructions

Click upload button in toolbar
Select one or more files
See the file(s) uploaded in the current selected folder.

# Todo
There is a lot to do until we have a final version of this feature:

* Show Progress
* Handle errors (filesize, file already exists...)
* Show Success/Error notifications
* Limit upload on client side (just for convenience. security is handled on server side)
* Check security on server side 
* Add correct html attributes to file input
...

I will work on some of the todos and create issues for them after this one is merged.
